core/metrics/src/lib.rs

```diff
@@ -3,3 +3,14 @@ mod metrics;
 mod service;
 
 pub use service::MetricsService;
+
+pub mod utils {
+    const CRYPTOCURRENCY_PRECISION: u64 = 1000000000;
```

Comment:
That's of course problematic if we allow other currencies than NGNT.

Reply:
no such plans afaik